### DISCUSSION TITLE
GentlemanDriverPlugin

### DISCUSSION THREAD

**Discussion by AGK1959** [sjdawson/GentlemanDriverPlugin#16] [General]
Hello, 
Have been reading your profile to upload into Simhub. I have a question. For ACC Sim racing will the json files be affected ? I have the lovely simdash and the profiles for my steering wheel and leds json files . Not sure if I should up load your plugin.

await your reply, Alex

**Comment by sjdawson** [comment]
Hi Alex,

My plugin doesn't (shouldn't) do anything to JSON files that already exist in SimHub. It creates it's own to store settings but that's it.

Which JSON files do you think will be affected by my plugin? And I can give you a direct answer in regard to those.

Cheers,

Steve.

**Comment by AGK1959** [comment]
Hi Steve

thanks your reply

have a look at the following website

danielnewmanracing.com/downloads.php

I have three files downloaded into Simhub for ACC and the json files in ACC folder.

I also play ETS and I have your dashboard set with your plugin great work.

Alex

**Comment by sjdawson** [comment]
So my plugin won't touch any of your existing LED profiles for those devices, it only adds additional properties that you can display on your dashboard. There is an (experimental) WLED implementation in there currently - but that also doesn't do anything with your existing LED profiles. 👍🏻

**Comment by AGK1959** [comment]
Thanks Steve
I am happy with the current dashboard so I won’t be uploading your profile.

Appreciate your honest reply.


Kind regards
Alex Katona
